products/jbrowse-web/src/SessionLoader.ts

```diff
@@ -566,6 +557,10 @@ const SessionLoader = types
                   }
                 }
                 if (sessionSnapshot) {
+                  if (!sessionPlugins) {
```

Comment:
this logic is somewhat complex: it says, if sessionSnapshot is defined, but sessionplugins is not, then re-set the session snapshot to itself

I would have to go through the logic to see in what cases this occurs but it may be worth adding a comment to explain if this is intended

otherwise, I am wondering if there is a better way to express this, so that the session plugins are already loaded somehow so this check is unneeded